### PR TITLE
Completed css-canvas-medical exercise.

### PR DESCRIPTION
**Pull request by ChorongShin**
Screen Shot 2022-01-29 at 9 30 33 PM
Screen Shot 2022-01-29 at 9 30 41 PM
Screen Shot 2022-01-29 at 9 30 50 PM

### CODE REVIEW

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Hey Stella! The text in the "Ready to learn more?" section is the wrong color, and you are missing an overlay on the hero image, the image in the output is a bit more faded than the image itself. I suggest looking back at the `absolute` portion of `css-positioning` for a reminder of how to do an overlay effectively/

**Comment by ChorongShin** [comment]
Screen Shot 2022-02-02 at 9 47 05 AM
Screen Shot 2022-02-02 at 9 47 12 AM

File: css-canvas-medical/styles.css [hunk 1]
```diff
@@ -0,0 +1,281 @@
...
+.hero-image {
+  width: 925px;
+  height: 458px;
+  background-image: url("images/hero.jpg");
+  background-size: cover;
+  background-repeat: no-repeat;
+  position: relative;
+  top: 50px;
+  background-position: center;
+  left: 239px;
```
Comment (codingcodymiller) [inline css-canvas-medical/styles.css:77]: **_woah_**, why does this need such a huge `left` value?

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

**Comment by ChorongShin** [comment]
_Users_chorongshin_repos_c0122-code-solutions_css-canvas-medical_index html (2)

**Review by codingcodymiller — APPROVED** [review]
Great work!

**Pull request merged**